CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2022-04-07
+
+### Added
+
+- new `license-checker/validate` command
+- new `license-checker/validate-copyleft` command
+
+### Breaking change
+
+- Remove `forbidden` parameter from `license-checker/validate-copyleft` job. Use `license-checker/validate` job if you want to customize the list of forbidden licenses.
+
+### Changed
+
+- Update default node executor version to use the `lts` tag
+- Upgrade `circleci/node` orb
+- Upload license report artifacts relative to `app-dir`
```

Comment:
Those are potential breaking changes. Not sure if we should move them. WDYT?

Reply:
I think it's fine like this. As you said they are more "potential" than real breaking changes.